src/Tools/MicroBuild/publish-assets.ps1

```diff
@@ -43,7 +43,7 @@ try
         "dev15.0.x" { } 
         "master" { } 
         "post-dev15" { } 
-        "features/refout" { }
+        "features_refout" { }
```

Comment:
The reason for this conversion was when we used to put branch names into the feed name, as MyGet doesn't allow the / in the feed names. Once we moved to a single feed line 39 could have been deleted. I won't complain if you just do that.